crates/vm/levm/tests/edge_case_tests.rs

```diff
@@ -1,3 +1,6 @@
+#![allow(clippy::indexing_slicing)]
+#![allow(clippy::unwrap_used)]
```

Comment:
I don't think these are needed. Checkout the `clippy.toml` and what we can write to avoid this.